classes/YARPP_Cache.php

```diff
@@ -456,10 +460,11 @@ function apply_filters_if_white($tag, $value) {
 	
 		if ( !isset($wp_filter['all']) )
 			$wp_current_filter[] = $tag;
-	
 		// Sort
 		if ( !isset( $merged_filters[ $tag ] ) ) {
-			ksort($wp_filter[$tag]);
```

Comment:
removed a warning when refreshing the YARPP cache when viewing a page for the first time. (This wasn't essential to adding it back to the plugin repo)